src/core/hiwire.c

```diff
@@ -366,6 +366,7 @@ EM_JS_NUM(bool, hiwire_is_function, (JsRef idobj), {
   // clang-format on
 });
 
+<<<<<<< HEAD
```

Comment:
@hoodmane Probably, you already noticed this, there's a merge issue.

Reply:
Thanks!